recipes-devtools/deepstream/deepstream-tao-apps_3.0.bb

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "NVIDIA Train, Adapt, and Optimize (TAO) Toolkit sample applications"
+LICENSE = "Apache-2.0"
```

Comment:
Looks like it's actually MIT, rather than Apache.

Reply:
Yep, you're right. I got it updated.